configs/default/DIAT-MAMBAF405US_I2C.config

```diff
@@ -0,0 +1,121 @@
+# Betaflight / STM32F405 (S405) 4.1.0 Oct 16 2019 / 11:57:16 (c37a7c91a) MSP API: 1.42
+
+board_name MAMBAF405US_I2C
+manufacturer_id DIAT
+
+# I/O
+resource LED 1 C15
+resource LED 2 C14
+resource BEEPER 1 C13
+set beeper_inversion = ON
+set beeper_od = OFF
+set osd_displayport_device = MAX7456
+
+# ACC/GYRO
+resource GYRO_EXTI 1 C04
+resource GYRO_CS 1 A04
+set gyro_1_bustype = SPI
+set gyro_1_spibus = 1
+set gyro_1_sensor_align = CW180
+set gyro_1_align_yaw = 1800
+
+# SPI
+resource SPI_SCK 1 A05
+resource SPI_MISO 1 A06
+resource SPI_MOSI 1 A07
+
+resource SPI_SCK 2 B13
+resource SPI_MISO 2 B14
+resource SPI_MOSI 2 B15
+
+resource SPI_SCK 3 C10
+resource SPI_MISO 3 C11
+resource SPI_MOSI 3 B05
+
+# FLASH
+resource FLASH_CS 1 A15
+set flash_spi_bus = 3
+
+# OSD
+resource OSD_CS 1 B12
+set max7456_spi_bus = 2
+
+# ADC
+resource ADC_BATT 1 C01
+resource ADC_RSSI 1 C02
+resource ADC_CURR 1 C03
+set adc_device = 3
+
+# SERIAL PORTS
+resource SERIAL_TX 1 B06
+resource SERIAL_RX 1 B07
+
+resource SERIAL_TX 2 A02
+resource SERIAL_RX 2 A03
+
+resource SERIAL_TX 3 B10
+resource SERIAL_RX 3 B11
+
+resource SERIAL_TX 4 A00
+resource SERIAL_RX 4 A01
+
+resource SERIAL_TX 5 C12
+resource SERIAL_RX 5 D02
+
+resource SERIAL_TX 6 C06
+resource SERIAL_RX 6 C07
+
+#I2C
+resource I2C_SCL 1 B08
+resource I2C_SDA 1 B09
+
+# TIMERS
+timer B09 AF3   # pin B09: TIM11 CH1 (AF3)
```

Comment:
Comments at the end of a line are not valid syntax in CLI. In CLI a comment is a line that starts with a `#`.

This syntax will work for now because parsing of command arguments stops after the last argument known to the firmware, but it will break if the command is ever extended to support additional optional arguments, so changing this so that the comments are on a line of their own before / after the command will make this more robust.

Reply:
Good point.   Fixed for new config and all Diatone configs.  Thanks for the catch.